e2e/playwright/pay.spec.ts

```diff
@@ -0,0 +1,359 @@
+
+  await clickButtonWithText("Yes");
+
+  await clickButtonWithText("No");
+
+  await clickButtonWithText("No");
+
+  await clickButtonWithText("No");
+
+  await clickButtonWithText("No, definitely not");
+
+  await page
+    .getByLabel("How many doors are being replaced with windows?")
+    .fill("1");
+
+  await page.getByTestId("continue-button").click();
+
+  await clickButtonWithText("No");
+
+  await clickButtonWithText("No");
+
+  await page.getByText("Private individual").click();
+  await page.getByTestId("continue-button").click();
+
+  await page.getByLabel("First name").fill("Test");
+  await page.getByTestId("continue-button").click();
+
+  await page.getByLabel("Last name").fill("Test");
+  await page.getByTestId("continue-button").click();
+
+  await page.getByLabel("Title").fill("Mx");
+  await page.getByTestId("continue-button").click();
+
+  await page.getByText("Yes", { exact: true }).click();
+  await page.getByTestId("continue-button").click();
+
+  await page.getByLabel("Email address").fill(TEST_EMAIL);
+  await page.getByTestId("continue-button").click();
+
+  await page.getByLabel("Telephone number").fill("99999");
+  await page.getByTestId("continue-button").click();
+
+  await page.getByText("Me, the applicant").click();
+  await page.getByTestId("continue-button").click();
+  await page.getByText("Sole owner").click();
+  await page.getByTestId("continue-button").click();
+
+  await uploadFileToInput(page)({ file: dummyImage });
+
+  await page.getByTestId("continue-button").click();
+
+  await uploadFileToInput(page)({ file: dummyImage });
+
+  await page.getByTestId("continue-button").click();
+
+  await clickButtonWithText("No");
+
+  await uploadFileToInput(page)({ file: dummyImage });
+
+  await page.getByTestId("continue-button").click();
+
+  await uploadFileToInput(page)({ file: dummyImage });
+
+  await page.getByTestId("continue-button").click();
+
+  await clickButtonWithText("No");
+
+  await clickButtonWithText("No");
+
+  await clickButtonWithText("No");
+
+  await clickButtonWithText("No");
+
+  await clickButtonWithText("No");
+
+  await page.getByText("I cannot provide any evidence").click();
+  await page.getByTestId("continue-button").click();
+
+  await page.getByText("5 hectares or less").click();
+  await page.getByTestId("continue-button").click();
+
+  await clickButtonWithText("No");
+
+  await clickButtonWithText("No");
+
+  await clickButtonWithText("No");
+
+  await clickButtonWithText("No");
+
+  await page.getByTestId("continue-button").click();
+
+  await page.getByText("None of the above apply to me").click();
+  await page.getByTestId("continue-button").click();
+
+  await page
+    .getByText("The information contained in this application is truthful")
+    .click();
+  await page.getByTestId("continue-button").click();
+
+  await page.getByText("South Buckinghamshire").click();
+  await page.getByTestId("continue-button").click();
```

Comment:
I agree with @jessicamcinchak here that in this instance we'd get more value from focusing on Pay and the various states which I think was the original intention.

What you have here though is great to see, and would likely be used elsewhere if we adopt Playwright, so it's still useful work which I think can / should be repurposed 👌

Reply:
I'm jotting down the code (commit) for posterity so we can utilise it as a starting point at some point in the future.